liana/method/sc/__init__.py

```diff
@@ -1,8 +1,10 @@
 from ._rank_aggregate import _rank_aggregate_meta
-from ._cellphonedb import cellphonedb
-from ._logfc import logfc
-from ._natmi import natmi
-from ._connectome import connectome
-from ._singlecellsignalr import singlecellsignalr
-from ._geometric_mean import geometric_mean
-from ._cellchat import cellchat
+from ._cellphonedb import cellphonedb, _cpdb_score
```

Comment:
same things applies to the scores, I would prefer to keep only a mean (we can just rename cpdb) & a product. Then have a third one (default) that takes additional information into consideration

Reply:
We then avoid adding flags, and complicating an already convoluted pipeline (pending refactoring) with redundant scoring functions